execution/record.go

```diff
@@ -29,6 +29,12 @@ func (id ID) Show() string {
 
 type RecordOption func(stream *Record)
 
+func WithClearUndo() RecordOption {
```

Comment:
I think maybe WithNoUndo() is a better name? Or maybe WithoutUndo()? The Clear is kinda confusing

Reply:
Changed to WithNoUndo